lib/.eslintrc.yaml

```diff
@@ -75,6 +75,8 @@ rules:
       message: Use `const { MessageEvent } = require('internal/worker/io');` instead of the global.
     - name: MessagePort
       message: Use `const { MessagePort } = require('internal/worker/io');` instead of the global.
+    - name: Navigator
+      message: Use `const navigator = require('internal/navigator');` instead of the global.
```

Comment:
```suggestion
    - name: Navigator
      message: Use `const { Navigator } = require('internal/navigator');` instead of the global.
    - name: navigator
      message: Use `const { navigator } = require('internal/navigator');` instead of the global.
```